spec/unit/provider/user/aix_spec.rb

```diff
@@ -217,4 +217,85 @@ def call_parse_password
       provider.create
     end
   end
+
+  describe '#delete' do
+    before(:each) do
+      allow(File).to receive(:realpath).and_call_original
+      allow(FileUtils).to receive(:remove_entry_secure).and_call_original
+
+      allow(provider.resource).to receive(:should).with(anything).and_return(nil)
+      allow(provider).to receive(:home).and_return(Dir.pwd)
```

Comment:
Why not set this a tmpdir, like `Dir.tmpdir` ?

Reply:
Great (a whole lot safer) idea! `File.realpath` needs a path that actually exists.